src/app/medInria/QSingleApplication/qtsingleapplication.cpp

```diff
@@ -168,47 +168,6 @@ QtSingleApplication::QtSingleApplication(const QString &appId, int &argc, char *
     sysInit(appId);
 }
 
-#if defined(Q_OS_X11)
-/*!
-  Special constructor for X11, ref. the documentation of
-  QApplication's corresponding constructor. The application identifier
-  will be QCoreApplication::applicationFilePath(). \a dpy, \a visual,
-  and \a cmap are passed on to the QApplication constructor.
-*/
-QtSingleApplication::QtSingleApplication(Display* dpy, Qt::HANDLE visual, Qt::HANDLE cmap)
```

Comment:
removed because these functions are not used anymore, and "Display" is not known

Reply:
Used: https://doc.qt.io/archives/qt-4.8/qapplication.html but not used anymore in https://doc.qt.io/qt-5/qapplication.html